libs/initScripts/local.js

```diff
@@ -171,6 +193,47 @@ const _registerCnode4 = async (audiusLibs, ethAccounts) => {
   await registerLocalService(audiusLibs2, spCreatorNodeType, creatorNodeEndpoint4, amountOfAuds)
 }
 
+// Account 1 - Cnode 1 Delegate Wallet Update
+const _updateCnode1DelegateOwnerWallet = async (ethAccounts) => {
```

Comment:
this could be refactored as a function that takes in the ethAccounts index so you don't need to duplicate this fn 4 times

Reply:
i would still have to pass ethAccounts here though which was my reason for not doing the above already